src/material.cpp

```diff
@@ -752,6 +752,10 @@ void Material::calculate_xs(Particle& p) const
   p.macro_xs_.absorption = 0.0;
   p.macro_xs_.fission = 0.0;
   p.macro_xs_.nu_fission = 0.0;
+  p.macro_xs_.prompt_nu_fission = 0.0;
+  p.macro_xs_.delayed_nu_fission.resize(8);
```

Comment:
I'm a little confused -- I thought you said this needs to be sized to the actual number of precursor groups, but here the size is effectively static.

Reply:
It's in a different part of the code, not included in this PR. When I use p.macro_xs_.delayed_nu_fission later in src/mgxs.cpp I have to resize the vector to exactly the number of actual delayed groups to avoid errors.

Reply:
Either way, people studying transients know immediately where the 8 comes from, but it would be good to define a `constexpr` variable like `MAX_PRECURSOR_GROUPS` rather than using the literal number 8 here. Just makes it more clear if some non-transient-experienced person has to work on this.

Reply:
I can easily use MAX_DELAYED_GROUPS here instead, no problem.